automated/testsuites/core-testsuite-001.yml

```diff
@@ -0,0 +1,121 @@
+device_type: RevPi_Core
+job_name: RevPi_Core - TestSuite 001
+priority: medium
+visibility: public
+tags:
+- config_core_01
+
+timeouts:
+  job:
+    minutes: 10
+  action:
+    minutes: 1
+  connection:
+    minutes: 2
+actions:
+- deploy:
+    timeout:
+      minutes: 4
+    to: ssh
+- boot:
+    method: ssh
+    connection: ssh
+    prompts: ["root@RevPi"]
+    timeout:
+      minutes: 2
+- test:
+    timeout:
+      minutes: 5
+    definitions:
+    - repository:
+        metadata:
+          format: Lava-Test Test Definition 1.0
+          name: worker-device-test
+          description: "Simple test if the test will be executed on given devices on a specific worker"
+        run:
+          steps:
+          - lava-test-case test1-pass --result pass
+          - lava-test-case test2-pass --result pass
+      from: inline
+      name: worker-device-test
+      path: inline/worker-device-test.yaml
+- test:
+    timeout:
+      minutes: 5
+    definitions:
+    - repository: https://github.com/RevolutionPi/LAVA-test-definitions.git
+      from: git
+      path: automated/linux/leds/test-leds1.yaml
+      name: test-leds1
+- test:
+    timeout:
+      minutes: 5
+    definitions:
+    - repository: https://github.com/RevolutionPi/LAVA-test-definitions.git
+      from: git
+      path: automated/linux/leds/test-leds2.yaml
+      name: test-leds2
+- test:
+    timeout:
+      minutes: 5
+    definitions:
+    - repository: https://github.com/RevolutionPi/LAVA-test-definitions.git
+      from: git
+      path: automated/linux/picontrol/test-pc-1.yaml
+      name: test-pc-1
+- test:
+    timeout:
+      minutes: 5
+    definitions:
+    - repository: https://github.com/RevolutionPi/LAVA-test-definitions.git
+      from: git
+      path: automated/linux/ethernet/test-eth-ethtool.yaml
+      name: test-eth-ethtool
+- test:
+    timeout:
+      minutes: 5
+    definitions:
+    - repository: https://github.com/RevolutionPi/LAVA-test-definitions.git
+      from: git
+      path: automated/linux/ethernet/test-eth-iperf3.yaml
+      name: test-eth-iperf3
+- test:
+    timeout:
+      minutes: 10
+    definitions:
+    - repository: https://github.com/RevolutionPi/LAVA-test-definitions.git
+      from: git
+      path: automated/linux/usb/test-usb-2.yaml
+      name: test-usb-2
+- test:
+    timeout:
+      minutes: 5
+    definitions:
+    - repository: https://github.com/RevolutionPi/LAVA-test-definitions.git
+      from: git
+      path: automated/linux/usb/test-usb-3.yaml
+      name: test-usb-3
+- test:
+    timeout:
+      minutes: 10
+    definitions:
+    - repository: https://github.com/RevolutionPi/LAVA-test-definitions.git
+      from: git
+      path: automated/linux/usb/test-usb-4.yaml
+      name: test-usb-4
+- test:
```

Comment:
Is there any reason, why almost all testcases have an own testspace but the last one with pibridge and pitest are added together?

Wouldn't it make sense to either group the testcases together by testtarget (e.g. USB) or have all of them in one single test?

Reply:
These tests use the same configuration, which is loaded with "config_loader". "config_loader" is not a test, it is used to configure the device.
I thought it would be a good idea to use both tests in the same testspace. 
But I will put the testcases together...